test/mocks/pair.ts

```diff
@@ -18,6 +18,8 @@ export const allPairs: IPairs = {
             decimals: 18,
         },
         price: "13.648737925900864974",
+        platformFee: "1%",
+        swapFee: "25%",
```

Comment:
in reverse order. swap fee is on every trade, platform fee is on LP earnings

Reply:
swapped values in test file